library/core/src/main/java/com/google/android/exoplayer2/text/cea/Cea608Decoder.java

```diff
@@ -230,6 +230,7 @@
   private boolean repeatableControlSet;
   private byte repeatableControlCc1;
   private byte repeatableControlCc2;
+  private boolean dataProcessedSinceLastRepeatableControl = true;
```

Comment:
Why does this default to true? Ditto for flush. We haven't seen any data at all in these cases, so I don't think it makes sense for it to be true. Comment may be obsolete if you can get rid of the field as per comment below.

Reply:
The very first "repeatable control" should not be handled as a repetition: default = true.